LibreNMS/Alert/Transport/Pagerduty.php

```diff
@@ -54,12 +55,18 @@ public function deliverAlert($obj, $opts)
      */
     public function contactPagerduty($obj, $config)
     {
+        $groups = Device::find($obj['device_id'])->groups;
+        foreach ($groups as $group)
```

Comment:
To just get an array of the group names this will work:
```php
\DeviceCache::get($obj['device_id'])->groups->pluck('name')->all();
```

Reply:
Thanks, just pushed an update, rolled the changes into a single line using DeviceCache as suggested.